pages/_app.js

```diff
@@ -61,6 +63,14 @@ export default function App({ Component, pageProps }) {
 
   const [hasCompletedStep5, setHasCompletedStep5] = useState(false);
 
+  // State to check the status if the user has selected values for all properties for a given distribution
+  // (e.g. bar plot)
+  const [hasCompletedDistProp, setHasCompletedDistProp] = useState(false);
```

Comment:
I took me a moment to think about, what "dist" is -> Perfect example to see that short notations should be avoided to keep the code readable for everyone. Better would be `hasCompletedDistributionProp`

Reply:
We have renamed this variable as hasCompletedDistributionProperties.